manifest/manifestlist/manifestlist_test.go

```diff
@@ -109,3 +110,105 @@ func TestManifestList(t *testing.T) {
 		}
 	}
 }
+
+// TODO (mikebrow): add annotations on the index and individual manifests
+var expectedOCIImageIndexSerialization = []byte(`{
```

Comment:
Thanks for checking! :)

I'm curious what other registry providers think about this. I'd really like to agree on some common behavior so we don't run into compatibility issues between registries.